test/unit/player.test.js

```diff
@@ -1428,59 +1428,22 @@ QUnit.test('Player#tech logs a warning when called without a safety argument', f
 });
 
 QUnit.test('player#reset loads the Html5 tech and then techCalls reset', function(assert) {
```

Comment:
I think we should keep this test as is and add a new test to test for the delayed reset functionality.